File: Teste Pratico HBSIS/HBSIS.Entity/Model/Usuario.cs
using HBSIS.Entity.Contracts.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HBSIS.Entity.Model
{
    public class Usuario : IEntity
    {
        public int Id_Usuario { get; set; }

        public string Nome { get; set; }

        public string Documento { get; set; }

        public string Telefone { get; set; }
    }
}

File: Teste Pratico HBSIS/HBSIS.WebApp/app/models/usuario.ts
export class UsuarioModel {
    public id_Usuario: number;
    public nome: string;
    public documento: string;
    public telefone: string;
}

export class Usuario extends UsuarioModel {
    constructor() {
        super();
        this.id_Usuario = 0;
        this.nome = '';
        this.documento = '';
        this.telefone = '';
    }
}